doc/source/data/faq.rst

```diff
@@ -0,0 +1,308 @@
+.. _datasets_faq:
+
+===
+FAQ
+===
+
+These are some Frequently Asked Questions that we've seen pop up for Ray Datasets.
+
+.. note::
+  For a general conceptual overview of Ray Datasets, see our
+  :ref:`Key Concepts docs <data_key_concepts>`.
+
+If you still have questions after reading this FAQ,  please reach out on
+`our Discourse <https://discuss.ray.io/>`__!
+
+.. contents::
+    :local:
+    :depth: 2
+
+
+What problems does Ray Datasets solve?
+======================================
+
+Ray Datasets aims to solve the problems of:
+
+1. Slow, unpipelined, and/or unscalable preprocessing and ingestion pipelines for
```

Comment:
Can we find a better way to express "unpipelined"? Depending on how favourably one wants to read this, this might be confused as "unpipelined [...] ingestion pipelines".

Also, the two bullet points are almost identical. Can we do something like:

Ray Datasets aims to solve the problem dealing with with slow or unscalable processing steps and ingestion pipelines for two primary use cases:

1. For **model training**, ...
2. For ...

Reply:
also, the way it's currently formulated does not quite make clear, what exactly the problem is.

Reply:
Yep I think that common extraction makes sense, or maybe making the end-problems of performance, cost, scalability more clear.